### PR TITLE
Change the emag text for the PTL and adds a sound

### PR DESCRIPTION
**Pull request by NibChocolateeny**
[Rework][Player Actions]

## About the PR 
If you emag the PTL, a small sound will play, and the player who emags it will get a more clear message hitning at what they just did (make it steal money with...negatife power output? power input? its crazy)

Also, for admins, theres a "demag" proc for the ptl now. :>

## Why's this needed? 
the money is being stolen! Now maybe someone can notice, maybe. (probably not!)

## Changelog 
```changelog
(u)NibChocolateeny
(+)Emagging the PTL makes a quiet sound, and has a clearer associated message
``

### CODE REVIEW

File: code/modules/power/pt_laser.dm [hunk 1]
```diff
@@ -76,7 +76,16 @@
 	src.emagged = TRUE
 	if (user)
 		src.add_fingerprint(user)
-		src.visible_message(SPAN_ALERT("[src.name] looks a little wonky, as [user] has messed with the polarity using an electromagnetic card!"))
+		playsound(src.loc, 'sound/machines/bweep.ogg', 10, 1)
```
Comment (TealSeer) [inline code/modules/power/pt_laser.dm:79]:
```suggestion
		playsound(src.loc, 'sound/machines/bweep.ogg', 10, TRUE)
```

File: code/modules/power/pt_laser.dm [hunk 2]
```diff
@@ -76,7 +76,16 @@
...
+		src.visible_message(SPAN_ALERT("The [src.name] quietly chirps 'FINANCE TRANSFER OVERRIDE' from some unseen speaker, then goes quiet."))
+	return 1
```
Comment (TealSeer) [inline code/modules/power/pt_laser.dm:81]:
```suggestion
	return TRUE
```

File: code/modules/power/pt_laser.dm [hunk 3]
```diff
@@ -76,7 +76,16 @@
...
+
+/obj/machinery/power/pt_laser/demag(var/mob/user)
+	if (!src.emagged)
+		return 0
```
Comment (TealSeer) [inline code/modules/power/pt_laser.dm:85]:
```suggestion
		return FALSE
```

File: code/modules/power/pt_laser.dm [hunk 4]
```diff
@@ -76,7 +76,16 @@
...
+	if (user)
+		user.show_text("You reset the [src.name]'s fund transfer protocols.", "blue")
+	src.emagged = FALSE
 	return 1
```
Comment (TealSeer) [inline code/modules/power/pt_laser.dm:89]:
```suggestion
	return TRUE
```

**Comment by Sbmhawk** [comment]
Doesn't this also put power back into the grid?

Swapping the polarity, or some mention of being able to buy power should be included imo.
Instead of just "fund transfer protocol"

**Comment by NibChocolateeny** [comment]
@Sbmhawk i dont think it can GIVE power to station, maybe jsut to itself? But it is more complicated than just money steal, yes. i changed the messages to hint at having negative outpit control when EMAGd

File: code/modules/power/pt_laser.dm [hunk 5]
```diff
@@ -76,8 +76,23 @@
 	src.emagged = TRUE
 	if (user)
 		src.add_fingerprint(user)
-		src.visible_message(SPAN_ALERT("[src.name] looks a little wonky, as [user] has messed with the polarity using an electromagnetic card!"))
-	return 1
+		playsound(src.loc, 'sound/machines/bweep.ogg', 10, TRUE)
+		src.visible_message(SPAN_ALERT("The [src.name] chirps 'OUTPUT CONTROLS UNLOCKED: INVERSE POLARITY ENABLED' \
```
Comment (Tarmunora) [inline code/modules/power/pt_laser.dm:80]:
```suggestion
		src.audible_message(SPAN_ALERT("The [src.name] chirps 'OUTPUT CONTROLS UNLOCKED: INVERSE POLARITY ENABLED' \
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**